docs/troubleshoot/verify-fingerprint.md

```diff
@@ -1,28 +1,35 @@
 # Verify Zowe runtime directory
 
-The Zowe runtime directory `ROOT_DIR` contains the code modules that make up Zowe.  If these are altered in any way the behavior of Zowe is unpredictable.  
+The Zowe runtime directory `ROOT_DIR` contains the code modules that make up Zowe.  If these code modules are altered in any way, the behavior of Zowe is unpredictable. To avoid this situation, Zowe provides a verify tool that you can use to check if the `ROOT_DIR` has been altered. The verify tool comprises a script file `zowe-verify-authenticity.sh` and the files it needs to check the release contents.
```

Comment:
Strictly speaking, the tool allows you to *detect* alterations, not to *avoid* them.  It does not give you a means to repair any alterations, merely to detect them.

Reply:
Changed to **"To check if the `ROOT_DIR` has been altered, Zowe provides a verify tool that comprises a script file `zowe-verify-authenticity.sh` and the files it needs to check the release contents."**